tests/test_utils.py

```diff
@@ -207,6 +213,59 @@ def test_quantiles(self):
             np.testing.assert_almost_equal(edges1, edges2, decimal=decimal)
 
 
+class ImageBasedTestCase(lsst.utils.tests.TestCase):
+    """A test case for testing sky position offsets for exposures."""
```

Comment:
Are the docstring and/or class name right?

Reply:
Agree, docstring looks wrong. I'm going to leave fixing these to @fred3m though to save having too many cooks here.